runtime/server/timeseries_api.go

```diff
@@ -432,7 +434,8 @@ func (s *Server) GenerateTimeSeries(ctx context.Context, request *runtimev1.Gene
 	if filter != "" {
 		filter = "WHERE " + filter
 	}
-	sql := `CREATE TEMPORARY TABLE _ts_ AS (
+	temporaryTableName := "_ts_" + uuid.New().String()
```

Comment:
This will output a UUID with dashes (see [docs](https://pkg.go.dev/github.com/google/uuid#UUID.String)), which feels fragile as a SQL identifier (though I think duckdb is supposed to support it). Let's drop the dashes to be safe, something like this should do the trick: `strings.ReplaceAll(uuid.New().String(), "-", "")`

Reply:
quotes handle that, see DuckDB ouput
```
D create table "k-b" (a int);
D select count(*) from "k-b";
┌──────────────┐
│ count_star() │
├──────────────┤
│ 0            │
└──────────────┘
```

Reply:
Let's still fix it to be safe (can do in another PR)